server/utilities/utilityFunctions.js

```diff
@@ -35,5 +35,32 @@ const decodeToken = (token) => {
     return decoded;
 }
 
+const checkQuestionType = (arr,type) => {
```

Comment:
these are not utiility functions they should be in the same file where they call.

Utility functions are the function which are generic and can be used by other functionality for there purpose

Reply:
Done